lib/archive_importer.rb

```diff
@@ -60,14 +61,14 @@ def set_auto_follow_back_aspect
     name = archive_hash["user"]["auto_follow_back_aspect"]
     return if name.nil?
 
-    aspect = user.aspects.find_by(name: name)
-    user.update(auto_follow_back_aspect: aspect) if aspect
+    aspect = @user.aspects.find_by(name: name)
```

Comment:
The usage of at (@) is the default for using accessors. There are problems writing to `user`, but accessing `@user` from outside. Rubocop is also whining.

Reply:
The warning was: RuboCop: Redundant `self` detected. [Style/RedundantSelf]
besides of that, I've tried to unify the self.user, user and @user notations in this file. The at-sign is IMHO the most clear statement that an instance variable is addressed. So I changed it to this notation. 
You may be right - "user" accesses the accessor, but the '@' makes it more clear what happens. I think, code style should rule that class variables should be used with the '@' - regardless if there is an accessor or not.

Reply:
We usually use the accessors (where they are available) and not the variable directly (maybe there are a few places where variables were access directly, but some have a special reason and others maybe are inconsistency from the past). If don't want to use the accessor and only use the variable directly, then we wouldn't even need to define the acccessor. Since the old code didn't have any rubocop warnings, I don't know why we would need to change it.

Reply:
If this is common 'diaspora' style, I won't like to change it.